imagestreams/redis-centos.json

```diff
@@ -9,59 +9,6 @@
   },
   "spec": {
     "tags": [
-      {
-        "annotations": {
-          "description": "Provides a Redis database on CentOS. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/redis-container/tree/master/5/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Redis available on OpenShift, including major version updates.",
-          "iconClass": "icon-redis",
-          "openshift.io/display-name": "Redis (Latest)",
-          "openshift.io/provider-display-name": "Red Hat, Inc.",
-          "tags": "redis"
-        },
-        "from": {
-          "kind": "ImageStreamTag",
-          "name": "5-el8"
-        },
-        "referencePolicy": {
-          "type": "Local"
-        },
-        "name": "latest"
```

Comment:
The latest part should be there and changed to something that is still supported. like 6-el8.